public/docs/_examples/cb-validation/ts/app/hero-form-model-binding.component.html

```diff
@@ -0,0 +1,75 @@
+ <!-- #docplaster -->
+ <!-- #docregion -->
+<div class="container">
+  <div [hidden]="submitted">
+    <h1>Hero Form (Model-Driven with Data Binding)</h1>
+    <form [ngFormModel]="heroForm" *ngIf="active" 
+          (ngSubmit)="onSubmit()">
+      <div class="form-group">
+        <!-- #docregion name-with-error-msg -->
+        <label for="name">Name</label>
+        <input type="text" class="form-control" 
+               [(ngModel)]="model.name"
+               ngControl="name"
+               [ngClass]="{'required': isRequired('name')}">
```

Comment:
I would doubt that the `aria-describedby` will be auto-rendered. I wonder if there should not a split between the standard validation attributes being applied and the user applying the ARIA. Otherwise there would be some ARIA properties added by the system and `aria-describedby` still being added by the user?